site/react/useAddPasskey.md

```diff
@@ -0,0 +1,41 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: useAddPasskey
+  - - meta
+    - name: description
+      content: An overview of the useAddPasskey hook
+  - - meta
+    - property: og:description
+      content: An overview of the useAddPasskey hook
+  - - meta
+    - name: twitter:title
+      content: useAddPasskey
+  - - meta
+    - name: twitter:description
+      content: An overview of the useAddPasskey hook
+---
+
+# useAddPasskey
+
+The `useAddPasskey` hook adds a new passkey to a user's Embedded Account.
+
+## Import
```

Comment:
we need a `## Params` section as well to highlight the optional MutationFn params. We can just link out to React Query for that. 

To avoid duplication of code in all the hooks, you can use a `.md` snippet as per https://vitepress.dev/guide/markdown#markdown-file-inclusion

Reply:
doing this below